OneBusAway/ui/themes/OBANavigationTitleView.h

```diff
@@ -0,0 +1,27 @@
+//
+//  OBANavigationTitleView.h
+//  org.onebusaway.iphone
+//
+//  Created by Cathy Oun on 5/13/17.
+//  Copyright © 2017 OneBusAway. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+@import MarqueeLabel;
+
+typedef NS_ENUM(NSInteger, OBAAppearanceNavBarTitleViewStyle){
+    OBAAppearanceNavBarTitleViewStyleDefault, // subtitle won't be displayed
+    OBAAppearanceNavBarTitleViewStyleSubtitle,
+};
+
+@interface OBANavigationTitleView : UIView
```

Comment:
Please wrap Objective-C header files with `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` in the future. More info: https://developer.apple.com/swift/blog/?id=25